.rubocop.yml

```diff
@@ -88,3 +88,6 @@ Minitest/AssertNil:
 
 Minitest/RefuteNil:
   Enabled: true
+
+Minitest/AssertEmpty:
+  Enabled: true
```

Comment:
That it was in the TODO file implies it was already enabled and this is redundant.